sdk/tables/azure-data-tables/azure/data/tables/_authentication.py

```diff
@@ -171,7 +174,7 @@ class BearerTokenChallengePolicy(BearerTokenCredentialPolicy):
 
     :param credential: The credential.
     :type credential: ~azure.core.TokenCredential
-    :param str scopes: Lets you specify the type of access needed.
+    :keyword str scopes: Lets you specify the type of access needed.
```

Comment:
It is not keyword arg.

Reply:
Pylint think it should be keyword, changed to disable the error.